books/system/doc/acl2-doc.lisp

```diff
@@ -132567,16 +132567,24 @@ introduction-to-the-tau-system) for more information about Tau.</dd>
  undo your changes; this will be explained further below.</p>
 
  <p>After making such changes, build an ACL2 executable image containing your
- modified code.  The next step is typically to create a new file, perhaps named
- after the main function(s) whose guards you want to verify, in directory
- @('books/system').  Community book @('books/system/too-many-ifs.lisp') is a
- good example.  Ultimately, this file should be a certifiable book that
- verifies the desired termination and guards.  Include this book in
- @('books/system/top.lisp').  Of course, you can instead add to some existing
- file in the same directory that is already included in
+ modified code, to test that the modified code builds.  The next step is
+ typically to create a new file, perhaps named after the main function(s) whose
+ guards you want to verify, in directory @('books/system').  Community book
+ @('books/system/too-many-ifs.lisp') is a good example.  Ultimately, this file
+ should be a certifiable book that verifies the desired termination and guards.
+ Include this book in @('books/system/top.lisp').  Of course, you can instead
+ add to some existing file in the same directory that is already included in
  @('books/system/top.lisp'), rather than creating a new book and including it
  there.</p>
 
+ <p>When verifying termination and guards of system functions in interactive
```

Comment:
Doesn't verify-termination attempt to verify guards iff there is an explicit guard (or type declaration, etc.)?

Reply:
Right; I don't think there's any "special way" that these files (books) are treated.  A link could be provided to :DOC set-verify-guards-eagerness, where it is explained that the default behavior is "yes if :guard, type or :stobjs is supplied".